src/plugins/GUITestBase/src/tests/common_scenarios/NIAID_pipelines/GTTestsNiaidPipelines.cpp

```diff
@@ -205,5 +205,83 @@ GUI_TEST_CLASS_DEFINITION(test_0004) {
     CHECK_SET_ERR(errors.size() == 0, "Unexpected errors");
 }
 
+GUI_TEST_CLASS_DEFINITION(test_0005) {
+    //1. Click Tools -> NGS data analysis -> Raw ChIP-Seq data processing... Choose Single-end
+    //2. Set "_common_data/fastq/lymph.fastq" and _common_data/fasta/DNA.fa as reads and reference in wizard
+    //3. Click "Next" several times and "Run"
+    //4. Wait for workflow finished
+    //Expected state: no errors
+
+    class custom : public CustomScenario {
+    public:
+        void run(HI::GUITestOpStatus &os) {
+            QWidget *dialog = QApplication::activeModalWidget();
+            CHECK_SET_ERR(dialog, "activeModalWidget is NULL");
+
+            QLineEdit *lineEdit1 = qobject_cast<QLineEdit *>(GTWidget::findWidget(os, "FASTQ files widget"));
```

Comment:
Use GTWidget::findExactWidget instead of dynamic/object casts. This way you do not need to check for nullptrs